io.sloeber.ui/src/io/sloeber/ui/monitor/internal/SerialListener.java

```diff
@@ -124,14 +126,13 @@ public void dispose() {
 		private String additionalSerialData = new String();
 
 		public synchronized void addData(String event) {
-			if (!event.isEmpty()) {
```

Comment:
I think this line should not be deleted?
Shouldn't we just restore SerialListerner?

Reply:
Sorry about that!